sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -425,8 +425,8 @@ def test_dofn_client_process_flush_called(self):
         test_client=client)
 
     fn.start_bundle()
-    fn.process(('project_id:dataset_id.table_id', {'month': 1}))
-    fn.process(('project_id:dataset_id.table_id', {'month': 2}))
+    fn.process(('project_id:dataset_id.table_id', ({'month': 1}, 'insertid1')))
```

Comment:
Can we add a test that explicitly checks that inserts IDs (as expected) get added ? (these tests also check for other things.

Also, optionally, it's good if we can somehow add a unit test that induces a failure in a direct runner based job (in a step before BQ sink)  to confirm that we do no regenerate the insert ID for the same element.

Reply:
That makes sense. I'll add a test.

Reply:
Added : D